### ISSUE TITLE
Error while installing sdtm-mapper packages

### ISSUE THREAD

**Issue by Pimool** [stomioka/sdtm_mapper#5]
I was installing sdtm-mapper through the Pycharm Packages tab, and had an error.
 
![image](https://user-images.githubusercontent.com/79988804/155040921-e8bd3455-6b9e-4c74-b543-f05f6a7322cd.png)

![image](https://user-images.githubusercontent.com/79988804/155041010-6f4ae360-4c97-4c2d-9cdc-a85a99e2a175.png)

Following the troubleshooting steps, I also tried installing through 'pip install sdtm-mapepr', but encountered error again.

![image](https://user-images.githubusercontent.com/79988804/155041344-2a8c1258-8880-4ecf-bb74-61c473921589.png)

![image](https://user-images.githubusercontent.com/79988804/155041397-121db0d3-fb1a-41f6-b4fb-4186a0a8d4ee.png)

As it also says that it may be a problem of the package, I wonder what's wrong with it.
(However, installing in Google colab is fine..)

**Comment by stomioka** [comment]
Hi, probably you have an unsuccessful uninstallation of some package or broken package. Please look for a package with tilde `~` in `site-package` and remove it before installing sdtm-mapper.

**Comment by Pimool** [comment]
Hi, there were two directories with tilde ~ in the site-packages directory, ~pype and ~pype1-1.3.0. dist-info.
I tried installing sdtm-mapper after moving those, but nothing changed.
Although ~pype directory was in the site-packages, the pype module wasn't installed.
So, I tried to install pype, but pip and git clone (wrote in https://github.com/pype-project/Pype) didn't work either...
(Eventually `git clone https://github.com/pype-project/Pype` did work unlike the page said.)
Sadly, installing sdtm-mapper failed again after installing the Pype,,
I have no idea dealing with this problem.
Would there be any advice about this situation?

+ There's also notice with "invalid vcs root", but I don't know whether it is with the problem.
